src/app/store/root.epic.ts

```diff
@@ -0,0 +1,10 @@
+import { combineEpics } from 'redux-observable';
+
+import { RootState, RootAction } from './root.reducer';
+import { clearProfile } from '../profiles/store/profile.epics';
+import { searchShows } from '../tvshows/store/tvshows.epics';
+
+export const rootEpic = combineEpics<RootAction, RootState>(
+  searchShows,
+  clearProfile
```

Comment:
Wouldn't this just be the `userProfile` to reflect a more accurate state key?

Reply:
Its not a state key, but side effect function that receives an Observable of Actions (and returns a new Observable of actions).   So the name "clearProfile" is so named because it deletes the profile data from the localStorage device.